### DISCUSSION TITLE
Slack plugin not working

### DISCUSSION THREAD

**Discussion by priya-dsp** [artilleryio/artillery#3299] [Artillery]
Hi @hassy , earlier Slack plugin was working and sending the report to Slack channel. From past 1 week, plugin is not sending any messages to the channel . Also, there is no error message showing with debug command. Could you please look into this once.

**Comment by bernardobridge** [comment]
Hello @priya-dsp,

There has been no change to the Slack plugin since 3 months ago, so it's unlikely this is due to a change on our side. Possibly something changed with your Slack channel setup? 

A few questions:

- What version of Artillery are you using, and was that version changed recently?
- What do you mean by "`debug` command"? How are you enabling debug?
- Are these Fargate/Lambda tests, or just local tests?

**Reply by priya-dsp** [reply]
Hi @bernardobridge . I'm using Artillery: 2.0.18 version. It's local test I ran and enabled debug logging with environment variable  set DEBUG=plugin:slack. No changes were made to Slack channel. However, in order to verify I created new channel and new webhook url but it didn't work.  

It's weird, I just noticed that it worked once and posted the message on slack channel. When I re-ran again, it didn't send message. It's like sometimes it works. Is this issue something related to Slack API causing this behavior ?

**Reply by priya-dsp** [reply]
@bernardobridge / @hassy

**Reply by hassy** [reply]
hi @priya-dsp 👋 Could you share the debug log from the Slack plugin with us please? it's hard to tell what could be going wrong with just the information we have so far. You can email the log to me on hveldstra@artillery.io
